src/start-server.js

```diff
@@ -19,9 +19,11 @@ export default () => {
       } else {
         console.log(`API server (API_PATH) is set to: ${app.apiPath}`);
       }
-      if (__DEVELOPMENT__ && app.devProxyToApiPath)
+      if (app.devProxyToApiPath)
         console.log(
-          `Using internal proxy: ${app.publicURL} -> ${app.devProxyToApiPath}`,
+          `Proxying API requests from ${app.publicURL}/++api++ to ${
```

Comment:
We might want to deprecate the legacy mode at some point too (since this might be false, if used).

Reply:
@sneridagh If legacyTraverse is true then the client does not add `/++api++` to URLs it fetches, but the devproxy is still running and would proxy a `/++api++` request to the same location. Right?